test/client_test.js

```diff
@@ -18,14 +18,32 @@ describe('Client', () => {
     expect(this.fhirClient.baseUrl).to.deep.equal(new URL(baseUrl));
   });
 
+  describe('invalid instance', () => {
+    const baseUrl = 'http://test.com';
+
+    it('responds to #authMetadata(), but returns false due to missing SMART OAuth URIs', async function () {
```

Comment:
It should be `#smartAuthMetadata`, and it looks like it returns an empty object rather than false.